dev-python/pymad/pymad-0.10.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
+DISTUTILS_USE_SETUPTOOLS=no
+
+inherit distutils-r1
+
+DESCRIPTION="Python wrapper for libmad MP3 decoding in python"
+HOMEPAGE="https://github.com/jaqx0r/pymad"
+SRC_URI="https://github.com/jaqx0r/${PN}/archive/version/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+DEPEND="media-libs/libmad"
+RDEPEND="${DEPEND}"
+S="${WORKDIR}/${PN}-version-${PV}"
+
+python_configure_all() {
+	set -- "${PYTHON}" config_unix.py --prefix "${EPREFIX}/usr"
+	echo "${@}"
+	"${@}" || die "Configuration failed"
+}
```

Comment:
is this needed at all?

Reply:
no, compared installs with and without, no difference